### PR TITLE
Fix the tags block delimiter

### PR DESCRIPTION
**Pull request by juanfra**
🎫 https://central.tri.be/issues/119393

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -216,6 +216,12 @@ Still not happy? Shoot us an email to support@theeventscalendar.com or tweet to
 
 == Changelog ==
 
+= [4.7.4] TBD =
+
+* Fix - Display the tags block delimiter in the editor [119393]
+
+= [4.7.3] TBD =
```
Comment (mitogh) [inline readme.txt]: I think this one should be 18.19 as `18.18` is `4.7.2`

Comment (juanfra) [reply readme.txt]: Thanks @mitogh! I've been asked to work directly on 19.01 so we can leave a clean Q for 18.19

Comment (mitogh) [reply readme.txt]: OMG! Didn't realize about that, so welcome to the first commit for the 2019 MR.

Comment (juanfra) [reply readme.txt]: 🙌

**Pull request merged**